config/locales/en.yml

```diff
@@ -19,3 +19,4 @@ en:
   topic: 'Topic'
   summary: 'Summary'
   show: 'Show'
+  subjects: 'Subjects'
```

Comment:
Please add all subjects name in normalize format like:
computer_science: 'Computer science'